static/app/views/issueList/groupListBody.tsx

```diff
@@ -108,15 +106,23 @@ function GroupList({
 
   return (
     <PanelBody>
-      {groupIds.map((id, index) => {
+      {(loading ? new Array(10).fill(0) : groupIds).map((id, index) => {
```

Comment:
Maybe make this 25 to match the limit we send in the request?

Reply:
👍🏼